hepcrawl/spiders/aps_spider.py

```diff
@@ -29,12 +31,17 @@ class APSSpider(Spider):
     Uses the APS REST API v2. See documentation here:
     http://harvest.aps.org/docs/harvest-api#endpoints
 
-    scrapy crawl APS -a 'from_date=2016-05-01' -a 'until_date=2016-05-15' -a 'set=openaccess'
+    Example usage:
+    .. code-block:: console
+
+        scrapy crawl APS -a 'from_date=2016-05-01' -a 'until_date=2016-05-15' -a 'set=openaccess'
+        scrapy crawl APS -a json_file=file://`pwd`/tests/responses/aps/aps_single_response.json -s "JSON_OUTPUT_DIR=tmp/"
+
```

Comment:
+1 for the examples, maybe document though the parameters too, you can do so with sphinx syntax and they will show nicely in the docs:
```
:param json_file: json file with the contents from blablabla
:type json_file: string
...
```